draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -0,0 +1,1568 @@
+   Blue.
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 6]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+       Customer Topology                   Customer Topology
+       Network Slice Blue                  Network Slice Red
+                            +---+         +---+      +---+
+                       -----|R3 |---   ---|R2 |------|R3 |
+                      /     +---+         +---+      +---+
+      +---+      +---+                                    \     +---+
+   ---|R1 |------|R2 |                                     -----|R4 |---
+      +---+      +---+                                          +---+
+
+                                 Customers
+   ---------------------------------------------------------------------
+                                 Provider
+
+        Customized Topology (Network Resouce Partions)
+        Provider Network with Virtual Devices
+
+        Network Slice Blue: VR1, VR3, VR5         +---+
+                                        ----------|VR5|------
+                                       /          +---+
+                    +---+         +---+
+              ------|VR1|---------|VR3|
+                    +---+         +---+
+              ------|VR2|---------|VR4|
+                    +---+         +---+
+                                       \          +---+
+                                        ----------|VR6|------
+        Network Slice Red: VR2, VR4, VR6          +---+
+
+                                 Virtual Devices
+   ---------------------------------------------------------------------
+                                 Physical Devices
+
+        Native Topology
+        Provider Network with Physical Devices
+                                                  +---+
+                                        ----------|R3 |------
+                                       /          +---+
+                    +---+         +---+
+              ======|R1 |=========|R2 |
+                    +---+         +---+
+                                       \          +---+
+                                        ----------|R4 |------
+                                                  +---+
+
+       Figure 2: Network Slicing Topologies for Virtualization
+
+
+
+
+
+
+Liu, et al.              Expires 13 August 2023                 [Page 7]
+
+Internet-Draft      Network Slice Topology Data Model      February 2023
+
+
+4.  YANG Model Overview
+
+   The following constructs and attributes are defined within the YANG
+   model:
+
+   *  Common attributes, which include a set of common attributes like
+      slice identifier, name, description, and names of customers who
+      use the slice.
+
+   *  Endpoints, which represent conceptual points of connection from a
+      customer device to the TNS.  An endpoint is mapped to specific
```

Comment:
About the definition of endpoint in the context of this draft -> Does it refer to the Customer endpoint, i.e. the SDP of the NBI YANG model, or does it refer to the endpoint in the provider side facing the SDP, that is, the peer-SAP of the NBI YANG model? I guess it should be the second, but not sure if we all have the same understanding.